### PR TITLE
WATERLOOP-31 add more functionality to the TextWithImage component

### PR DESCRIPTION
**Pull request by jmawloo**
This could potentially deprecate the need for the SimpleFeatures component since the added functionalities allow this component to behave like that component. :)

### CODE REVIEW

File: waterloop-site/src/theme/global.scss [hunk 1]
```diff
@@ -200,13 +200,64 @@ h1 {
   text-align: justify;
   width: 50%;
   height: 100%;
-  padding-right: 3%;
   display: flex;
   flex-direction: column;
-  margin-bottom: 3em;
+}
```
Comment (jmawloo) [inline waterloop-site/src/theme/global.scss:205]: Rearranged CSS selectors to allow selectors in media query to overwrite these ones.

**Comment by willakong** [comment]
The responsiveness of the component works well. However, in mobile view, should there be more padding in accordance with the design specifications?

image

**Comment by jmawloo** [comment]
@willakong in regards to your comment on padding, it's in my personal opinion that there should be a universal padding introduced by the parent containing the TextWithImage component. I'll ask Jerin and Lynn about this.

**Comment by jmawloo** [comment]
**TODO for Jeff:** Refactor code so that images can be pulled from static folder

**Pull request merged**